docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -46,7 +46,7 @@ Attributes
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``size``                    | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``style``                   | N                               |                                                    |
+| ``style``                   | Y                               |                                                    |
```

Comment:
Should we add a note here saying this defaults to pandas?

Reply:
> Should we add a note here saying this defaults to pandas?

+1, good idea